frontend/src/Pages/Students.tsx

```diff
@@ -0,0 +1,340 @@
+                                                            editUserModal.current?.showModal();
+                                                        }}
+                                                    />
+                                                </div>
+                                                <div
+                                                    className="tooltip"
+                                                    data-tip="Reset Password"
+                                                >
+                                                    <ArrowPathRoundedSquareIcon
+                                                        className="h-4"
+                                                        onClick={() => {
+                                                            setTargetUser(user);
+                                                            resetUserPasswordModal.current?.showModal();
+                                                        }}
+                                                    />
+                                                </div>
+                                                <div
+                                                    className="tooltip"
+                                                    data-tip="Delete User"
+                                                >
+                                                    <TrashIcon
+                                                        className="h-4"
+                                                        onClick={() => {
+                                                            setTargetUser(user);
+                                                            deleteUserModal.current?.showModal();
+                                                        }}
+                                                    />
+                                                </div>
+                                            </div>
+                                        </td>
+                                    </tr>
+                                );
+                            })}
+                    </tbody>
+                </table>
+                {!isLoading && !error && userData.length != 0 && (
+                    <Pagination meta={data.meta} setPage={setPageQuery} />
+                )}
+                {error && (
+                    <span className="text-center text-error">
+                        Failed to load users.
+                    </span>
+                )}
+                {!isLoading && !error && userData.length == 0 && (
+                    <span className="text-center text-warning">No results</span>
+                )}
+            </div>
+
+            <Modal
+                ref={addUserModal}
+                type={ModalType.Add}
+                item="User"
+                form={<AddUserForm onSuccess={onAddUserSuccess} />}
+            />
```

Comment:
@PThorpe92 @jtucholski Something to think about - this `AddUserForm` currently has the ability to select the type of user you are adding.. should we remove this to avoid users from adding admin from the student page and vise versa? ie we can just default send the role to the backend for now? In the future when we have varying roles of admin this field would be relevant so not sure if we just want to keep it for now? Let me know your thoughts

Reply:
That is a good point. I think for now the role can just be included to the backend, plus it will default to student if not provided